shardingsphere-infra/shardingsphere-infra-common/src/main/java/org/apache/shardingsphere/infra/yaml/config/swapper/YamlDataSourceConfigurationSwapper.java

```diff
@@ -67,6 +71,9 @@ public DataSourceConfiguration swapToDataSourceConfiguration(final Map<String, O
      */
     public Map<String, Object> swapToMap(final DataSourceConfiguration dataSourceConfig) {
         Map<String, Object> result = new HashMap<>(dataSourceConfig.getProps());
+        if (dataSourceConfig.getCustomPoolProps().isEmpty()) {
```

Comment:
Why put when `dataSourceConfig.getCustomPoolProps()` is empty?

Reply:
Thanks , It was a mistake submitted in ffc7a212a5d3084de3b212fe7a5f011a969cd831